### PR TITLE
In Pyramid module, raise tween above EXCVIEW.

### PR DESCRIPTION
**Pull request by luhn**
Currently the Rollbar tween is put below the EXCVIEW.  It's desirable to
put the Rollbar tween above EXCVIEW to catch errors that might occur
within EXCVIEW.  Also, in order to catch errors in tweens that are
placed above EXCVIEW (such as `pyramid_tm`), Rollbar must support being
placed above EXCVIEW as well.

This commit modifies the Rollbar tween to extract an exception from
`request.exception` and `request.exc_info` if available.  It also moves
the tween to above EXCVIEW.

### CODE REVIEW

**Comment by coryvirok** [comment]
Sorry for the long response time on this but this LGTM. @rokob?

**Comment by luhn** [comment]
Ping.  Anything more needed to have this PR merged?

File: rollbar/contrib/pyramid/__init__.py [hunk 1]
```diff
@@ -40,31 +49,25 @@ def parse_settings(settings):
...
     def rollbar_tween(request):
         # for testing out the integration
         try:
             if (settings.get('allow_test', 'true') == 'true' and
                     request.GET.get('pyramid_rollbar_test') == 'true'):
                 try:
                     raise Exception("pyramid_rollbar test exception")
-                except:
-                    handle_error(settings, request)
+                except Exception as exc:
```
Comment (rokob) [inline rollbar/contrib/pyramid/__init__.py]: Technically, `except Exception as exc:` is different from `except:` but I am not sure we wanted to catch those special cases (BaseException, SystemExit, KeyboardInterrupt, etc) in the first place.

**Comment by rokob** [comment]
LGTM as well

**Pull request merged**